src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -717,11 +717,13 @@ export default class AircraftModel {
      * @return {string}
      */
     getRadioWeightClass() {
```

Comment:
isn't this implemented already in the `AircraftTypeModel`?

Reply:
I don't think so but I feel like this method should be part of `AircraftTypeDefinitionModel`

Reply:
It was not. I agree I suppose, but the other `.getRadio()` stuff is in the AircraftModel, right next to this method, so it's okay to leave there for now I think.

Reply:
moved to AircraftTypeDefinitionModel and added tests